rero_ils/modules/monitoring/cli.py

```
@@ -128,11 +127,14 @@ def es_indices():
-    """Displays redis info."""
-    url = current_app.config.get("ACCOUNTS_SESSION_REDIS_URL", "redis://localhost:6379")
-    redis = Redis.from_url(url)
-    for key, value in redis.info().items():
-        click.echo(f"{key:<33}: {value}")
+    """Displays redis info for all configured instances."""
+    for name, client in current_app.extensions.get("rero_ils_redis_instances", {}).items():
+        click.secho(f"\n[{name}]", fg="green")
+        try:
+            for key, value in client.info().items():
+                click.echo(f"  {key:<33}: {value}")
+        except Exception as err:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)